feat: Add safari as supported platform

ptbrowne opened the pull request:
- Link in available platforms card
- Link in "unsupported platforms" info
- Link in installation page to mac app store + instructions

The unsupported platforms part was removed in a former PR e3e159d442fb351da28dc9b656799eaa784aaf9b but it was important so I
reintroduced it here.

ptbrowne committed:
feat: Add safari as supported platform

- Link in available platforms card
- Link in "unsupported platforms" info
- Link in installation page to mac app store + instructions

ptbrowne committed:
fix: Use origin as default deploy repository

ptbrowne committed:
chore: Bump to 0.3.7

Crash-- commented:
Do you plan to wait for the mas validation before merging the PR?

Crash-- commented:
> The unsupported platforms part was removed in a former PR e3e159d but it was important so I
reintroduced it here. 

Did I review that ? I don't remember :o

ptbrowne committed:
fix: Use snarkdown on step 1

For safari, we need the "Get" to be bold

ptbrowne commented:
> Do you plan to wait for the mas validation before merging the PR?

Not necessarily, we can wait to publish it in prod though.

ptbrowne commented:
> Did I review that ? I don't remember :o

No, bad me, I wrongly thought that this being just a deletion, was ok to do without review. I was wrong ;)

Crash-- commented on src/supportedPlatforms.js:
```diff
@@ -9,6 +9,10 @@ const supportedPlatforms = {
+  // safari: {
+  //   label: 'Safari',
+  //   storeUrl: 'macappstore://itunes.apple.com/app/id1504487449?mt=12'
+  // }
```
can we uncomment these lines and use a flag? Like that we can deploy the app in production and just set the flag value when the app is validated?

ptbrowne replied:
Yes but I have to refactor the code since the code is executed at module level.

ptbrowne replied:
Done in 7edfeca

ptbrowne committed:
refactor: Use a flag for safari app availability

Since supportedPlatforms were declared at module level, we had to
refactor to introduce a function call, to be able to lazily evaluate
supportedPlatforms value
